content/en/Platform Deep Dive/Cobalt Account/account-settings.md

```diff
@@ -66,42 +55,47 @@ To manage two-factor authentication (2FA) settings for your account, select **Se
 
 #### Enable Two-Factor Authentication
 
-Even if your organization hasn't enforced 2FA for all users, we recommend that you enable it to enhance your account security. If you're using [SAML SSO](/getting-started/sign-in/#saml-sso) to sign in, you don't need to turn on 2FA.
+If you're **signing in with your email and password**, you can enable two-factor authentication (2FA) to protect your account. 2FA adds another layer of security to your account by requiring more than just a password to sign in.
 
-![Configure 2FA for your account](/deepdive/SecuritySettings.png "Two-factor authentication page")
+- Even if your organization doesn't [enforce 2FA](/platform-deep-dive/collaboration/organization/organization-settings/enforce-2fa/) for all users, we recommend that you enable it to enhance your account security.
+- If you're signing in through single sign-on, 2FA doesn't apply. This includes [SAML SSO](/getting-started/sign-in/#saml-sso) and Google authentication.
```

Comment:
Instead of Google authentication, should we say OAuth

Reply:
Changed this to _Google authentication (OAuth 2.0)_ in several files.